### PR TITLE
Minor content change suggestions

### PR DESCRIPTION
**Pull request by smaranjitghose**
## What type of PR is this?

- [ ] Feature
- [ ] Bug Fix
- [ ] Refactor
- [X] Documentation Update

## Description

Very minimal changes to ReadMe content

## Added unit tests?

- [ ] No, I want help in writing tests.
- [ ] No, I want someone else to write tests.
- [X] No, The addition does not need tests.
- [ ] Yes

 optional

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +30,17 @@ Changelogs are maintained in [CHANGELOG.md](CHANGELOG.md)
 ### 🤗 Local Installation & Contributing
 
 - Fork [abelljs/abell](https://github.com/abelljs/abell)
-- `git clone https://github.com/:github-username/abell`
-- `cd abell`
+- `git clone https://github.com/:github-username/abell` ⏬
```
Comment (saurabhdaware) [inline README.md:33]: Usually, people double click to select a line to copy, if we add this icon at the end and someone selects the line, they will get `git clone https://github.com/:github-username/abell arrow_double_down` in the clipboard. So maybe we can remove that icon from there.

Also, I realized even in the lines below I've added text that is being copied on selecting a line.

One solution I can think of is to have all these commands in a single block and write comments in `#`

```sh
git clone 
npm install
npm link # This text will be copied but terminal understands that this is a comment and ignore this text in the command
```

Something like this ^^. Will you be interested in making these changes?

Comment (smaranjitghose) [reply README.md:33]: Sure

**Comment by saurabhdaware** [comment]
Merged in #50

**Comment by saurabhdaware** [comment]
Thank you and you can usually push commits to the same branch and they get added to the Pull Request so you don't have to create a new PR with changes :)

**Pull request closed without merging**